.env

```
@@ -27,4 +27,11 @@ JWT_SECRET=mySecretKey12345678901234567890123456789012345678901234567890
 JWT_EXPIRATION=86400000
 
 # Inter-service secret
-INTER_SERVICE_SECRET=inter-service-secret-key-123456789
\ No newline at end of file
+INTER_SERVICE_SECRET=inter-service-secret-key-123456789
+
+# Khóa 1: Dùng 1 lần để deploy contract (Lấy từ tài khoản INDEX 0)
+DEPLOYER_PRIVATE_KEY=cfa19f3e41ec1cf77c88f6122835c049163b69c9c3ca02e4ba1dba9ff196cd23 
```

dotenv-linter (4.0.0)

[warning] [TrailingWhitespace] Trailing whitespace detected

(TrailingWhitespace)